### PR TITLE
Samoan Translation v1

### PR DESCRIPTION
**Pull request by gpoliko**

### CODE REVIEW

**Comment by wseagar** [comment]
Haven't been across the translation stuff too much but how about we order the list of languages by how commonly they are used in New Zealand?

IMO we have a fairly large Samoan community so it would nice to put our pacific languages + Chinese at the top before less used languages like Russian and Spanish. Any thoughts @olafwrieden ?

**Comment by olafwrieden** [comment]
@wseagar I swear you read my mind... I am literally doing that right now :D

File: src/translations/locales/common_sa-SA.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+      "25km": "Totonu ole 25km",
+      "50km": "Totonu ole 50km",
+      "100km": "Totonu ole 100km"
+    }
+  },
+  "walkins": {
+    "sectionHeader": "Ulufale atu poo le tala'ao atu i lau ta'avale <0>- Matala ile asō</0>",
+    "openString": "Taimi matala - {{openTimeString}}",
+    "cancellationNotice": {
+      "info": "Faamolemole e aua le faia faalua se avanoa",
```
Comment (tausani-ah-chong) [inline src/translations/locales/common_sa-SA.json:42]: Minor - Just for consistency I reckon we add the glottal stops to this sentence. There a couple other sentences I can see too. What do you think?

**Comment by gpoliko** [comment]
Great point @wseagar!

File: src/translations/locales/common_sa-SA.json (hunk 1 above)
Comment (gpoliko) [reply src/translations/locales/common_sa-SA.json:42]: Yeah for sure! I agree 100% on getting the glottal stops in. Happy for you to correct those if you wanted.

File: src/translations/resources.ts [hunk 2]
```diff
@@ -19,6 +20,21 @@ const languages: Language[] = [
     label: "English",
     code: "en-NZ",
   },
+  {
+    common: common_sa_SA,
+    label: "Gagaga Samoa",
```
Comment (tausani-ah-chong) [inline src/translations/resources.ts:25]: Minor - Just a fix for spelling "Gagana"

**Comment by gpoliko** [comment]
@olafwrieden I believe @tausani-ah-chong will review and make the final sign off on this translation.

**Comment by gpoliko** [comment]
@tausani-ah-chong could you review the changes when you get a chance please. There are a few glottal stops that I've kept in as they're pretty much required otherwise the word would look out of sorts.

File: src/translations/locales/common_sa-SA.json [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    "makeABooking": "Faamauina lou avanoa",
+    "kmAway": "{{distance}} kilomita le alu",
+    "lastUpdated": "Taimi mulimuli ga fa'afouina ai - {{updatedAt}}",
+    "cancel": "Fa'aleaogaina",
+    "website": "'Upega tafa'ilagi"
+  },
+  "navigation": {
+    "about": "About",
+    "contact": "Contact",
+    "getInvolved": "Get Involved",
```
Comment (tausani-ah-chong) [inline src/translations/locales/common_sa-SA.json:18]: Do we want to translate these also?

**Review by tausani-ah-chong — APPROVED** [review]
Only have one minor comment. Other than that, looks seki!

**Review by tausani-ah-chong — APPROVED** [review]

File: src/translations/locales/common_sa-SA.json (hunk 3 above)
Comment (olafwrieden) [reply src/translations/locales/common_sa-SA.json:18]: @tausani-ah-chong if there is a translation for it that you would typically see on websites than yes please! Let me know when you're all done

**Review by olafwrieden — APPROVED** [review]

**Pull request merged**